lambdas/retrieve-cri-credential/src/main/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandler.java

```diff
@@ -222,17 +222,6 @@ public Map<String, Object> handleRequest(Map<String, String> input, Context cont
                 | CiPostMitigationsException e) {
             updateVisitedCredentials(
                     ipvSessionItem, credentialIssuerId, null, false, OAuth2Error.SERVER_ERROR_CODE);
-            try {
-                sendCriResRetrievedAuditEvent(
-                        auditEventUser,
-                        credentialIssuerId,
-                        CriResourceRetrievedType.ERROR.getType());
-            } catch (SqsException ex) {
-                LogHelper.logErrorMessage(
-                        "Failed to send cri_response_retrieve audit event to SQS queue.",
-                        ex.getMessage());
-            }
```

Comment:
This means we're no longer sending a CRI resource retrieved error event - do you know when we're supposed to be sending one?

Reply:
I think it should be when the CRI response return as vc status as error, which currently it doesn't.

Reply:
Ok - if you don't think we'll need to send error events for now, then we should probably just remove it as a possible status (YAGNI).

Reply:
Done and updated TxMA document.